wordpress-mobile/WordPress-Android
# Mobile stories block - part 5: empty block editing

This PR builds on top of #13103

This adds the ability to edit an empty story block just added via the block picker, as shown in the gif below.

![emptyblockediting](https://user-images.githubusercontent.com/6597771/96464175-e4ebf880-11fd-11eb-9007-120ae4279e2e.gif)


To test:
1. create a post (either regular or a story post)
2. open in in gutenberg
3. add a Story block
4. tap on the empty Story block to trigger the StoryComposer camera capture mode to add a frist slide to your Story.
5. when one, tap on the arrow button to the top-right
6. observe the story slides get saved in the block.

Related PRs:
Gutenberg-mobile https://github.com/wordpress-mobile/gutenberg-mobile/pull/2747
Jetpack https://github.com/Automattic/jetpack/pull/17539


PR submission checklist:
